README.md

```diff
@@ -7,7 +7,7 @@ Python plugin for [asdf](https://github.com/asdf-vm/asdf) version manager
 ## Install
 
 ```
-asdf plugin-add python
+asdf plugin add python
```

Comment:
@adaao The asdf documentation also encourage to use the complete path to add the plugins, I think we can change to something like this:
```suggestion
asdf plugin add python https://github.com/asdf-community/asdf-python.git
```
I tested this script here and works just fine